### PR TITLE
[ENH] - Add axis inverting helper function and apply to spatial plots

### PR DESCRIPTION
**Pull request by TomDonoghue**
This PR adds a helper function for inverting plot axes, and then links this in to the spatial plots, allowing for directly controlling the plot orientation for spatial plots. 

Note that in these cases, inverting plot axes is functionally equivalent to flipping a 2d data (where flipping the 2d array up/down is the same as inverting the y axis, and flipping a 2d array left/right is equivalent to inverting the x axis).

Note: this addition stems from a discussion (with @maestapereira, I think) about adding helper utilities for flipping position plot orientations, since sometimes the original orientation of the position data isn't the same as how we want to plot it.

### CODE REVIEW

File: spiketools/plts/utils.py [hunk 1]
```diff
@@ -190,3 +190,26 @@ def get_grid_subplot(grid, row, col, **plt_kwargs):
...
+    ----------
+    ax : Axes
+        Axis object to update.
+    invert : {'x', 'y', 'both'} or None
+        How to invert the plot axes, inverting the x, y, or both axes.
+
+    Notes
+    -----
+    Note that inverting axes is equivalent to flipping the data, specifically:
+    Flipping up/down is equivalent to inverting the  y-axis.
```
Comment (maestapereira) [inline spiketools/plts/utils.py:208]: Extra space between "the" and "y-axis".

**Review by maestapereira — APPROVED** [review]
Great addition, thanks!

**Review by claire98han — APPROVED** [review]
Looks great!

File: spiketools/plts/utils.py (hunk 1 above)
Comment (TomDonoghue) [reply spiketools/plts/utils.py]: good catch! Fixed!

**Pull request merged**